subiquity/cmd/tui.py

```diff
@@ -87,6 +87,8 @@ def main():
         opts, unknown = parser.parse_known_args(args)
         if opts.socket is None:
             os.makedirs('.subiquity', exist_ok=True)
+            if os.path.exists('.subiquity/run/subiquity/server-state'):
+                os.unlink('.subiquity/run/subiquity/server-state')
```

Comment:
This must also be done in system_setup/cmd/tui.py

Reply:
oh yes